app.py

```diff
@@ -0,0 +1,50 @@
+from flask import Flask, render_template, jsonify, request
+from transbank import onepay
+from transbank.onepay.cart import ShoppingCart, Item
+from transbank.onepay.transaction import Transaction, Channel
+from transbank.onepay.refund import Refund
+
+app = Flask(__name__)
+
+onepay.api_key = "dKVhq1WGt_XapIYirTXNyUKoWTDFfxaEV63-O5jcsdw"
```

Comment:
This should be needed. The sdk should default to a api key that works in the test environment (like it does in PHP, Java and dotnet) cc @eugeniolopezramos